### PR TITLE
Fix Swagger Render On Remote

### PR DESCRIPTION
**Pull request by alfredoihernandez**
# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [x] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [ ] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR
Fix loading of swagger on dev endpoint. Ensure local swagger and remote UI continues to work. After this PR most remote endpoint calls on dev will be prefixed with /api/api

## Does this introduce a breaking change

- [ ] YES
- [ ] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/829

### CODE REVIEW

**Review by kforeverisback — APPROVED** [review]
Tested in a local k3d cluster.

Swagger URL prefix changes look good.

**Comment by kforeverisback** [comment]
Can you sync with @joaquinrz to make sure autogitops yaml changes here is sufficient to for our dev clusters?

File: src/LodeRunner.API/src/Core/Startup.cs [hunk 1]
```diff
@@ -114,22 +114,30 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env)
...
-            app.Use(async (context, next) =>
-            {
-                context.Response.Headers.Add("X-B3-TraceId", Activity.Current.Context.TraceId.ToString());
-                context.Response.Headers.Add("X-B3-SpanId", Activity.Current.Context.SpanId.ToString());
-
-                // call next middleware handler
-                await next().ConfigureAwait(false);
-            });
+
+            app.Use(async (context, next) =>
```
Comment (kforeverisback) [inline src/LodeRunner.API/src/Core/Startup.cs:118]: This diff looks identical.
Wondering if there are any whitespace changes here.

Comment (alfredoihernandez) [reply src/LodeRunner.API/src/Core/Startup.cs:118]: EOL changes

**Pull request merged**